ldcbench.http-node/src/main/java/org/dice_research/ldcbench/nodes/http/simple/dump/comp/ZipArchiver.java

```diff
@@ -0,0 +1,34 @@
+package org.dice_research.ldcbench.nodes.http.simple.dump.comp;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import org.apache.commons.compress.archivers.zip.ZipArchiveOutputStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ZipArchiver extends Archiver {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ZipArchiver.class);
+
+    private static final String MEDIA_TYPE = "application/zip";
+    private static final String FILE_NAME_EXTENSION = ".zip";
+    ZipArchiveOutputStream zout;
```

Comment:
Class attributes should have an explicit visibility defined. I would suggest `private` or `protected`.